doc/source/using.rst

```diff
@@ -13,6 +13,12 @@ source and the rendering template. It requires two parameters.
    ``template``
       The name of a template file on the Sphinx template search path.
 
+   ``key``
+      **Optional**: A specific data key to pass to the template, so you can render a subset of the data.
+
+   ``include_context``
+      **Optional**: A flag that includes the ``html_context`` from your normal build in the _template_ rendering context.
```

Comment:
This is cool. Is there any reason for it to be optional? Were you worried about namespace collisions?